### PR TITLE
Add implementation of Area of Triangle by using Heron's Formula in C++

### PR DESCRIPTION
**Pull request by piyush26c**
**Changes:**
The code was rewritten according to appropriate C++ style coding.

### CODE REVIEW

**Comment by agzuniverse** [comment]
I think the current area of triangle program implementation is fine. Also, when you are rewriting any code, don't create a new file just for the rewrite, make the changes in the same file and send a PR. This avoids needless duplicates.

File: code/computational_geometry/src/area_of_triangle/area_of_triangle_new.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{
+    double ta, tb, tc;
+    std::cout << "\nEnter the length of side-1 : ";
+    std::cin >> ta;
+    std::cout << "\nEnter the length of side-2 : ";
+    std::cin >> tb;
+    std::cout << "\nEnter the length of side-3 : ";
+    std::cin >> tc;
+    AreaOfTriangle a(ta, tb, tc);
+    if(a.calculateArea() == 0.0)
```
Comment (Vaibhavi24) [inline code/computational_geometry/src/area_of_triangle/area_of_triangle_new.cpp]: Add space after if.

**Comment by Vaibhavi24** [comment]
@agzuniverse I guess this code is implementing Heron's formula while the earlier exisiting code uses determinants. Pls confirm @piyush26c

File: code/computational_geometry/src/area_of_triangle/area_of_triangle_new.cpp (hunk 1 above)
Comment (piyush26c) [reply code/computational_geometry/src/area_of_triangle/area_of_triangle_new.cpp]: @Vaibhavi24 Yes this implementation implements Heron's Formula, Are we able to merge this implementation in existing one?

Comment (piyush26c) [reply code/computational_geometry/src/area_of_triangle/area_of_triangle_new.cpp]: > Add space after if.

Space added after if.

**Comment by piyush26c** [comment]
> @agzuniverse I guess this code is implementing Heron's formula while the earlier existing code uses determinants. Pls confirm @piyush26c

@Vaibhavi24 Yes this implementation implements Heron's Formula, Are we able to merge this implementation in existing one?

**Comment by piyush26c** [comment]
> Neat implementation!

Thank You!

File: code/computational_geometry/src/area_of_triangle/area_of_triangle_new.cpp (hunk 1 above)
Comment (Vaibhavi24) [reply code/computational_geometry/src/area_of_triangle/area_of_triangle_new.cpp]: I meant like on Line #25, 'if (' not 'if( '

**Comment by piyush26c** [comment]
> Fix all spacing issues. Refer c++ style guide. I meant like on Line #25, 'if (' not 'if( '

Fixed the issues, please review it.

**Comment by piyush26c** [comment]
I had done all the changes required in following pull request : 
Add implementation of Conversion of Infix to Postfix using Stack in C++ #4038
Please review it.

**Comment by Vaibhavi24** [comment]
@arnavb Pls merge this one

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**
positive_opengenus

**Pull request merged**